doc/API.md

```diff
@@ -608,6 +613,120 @@ In axe-core v1 the main method for axe was `axe.a11yCheck()`. This method was re
 - .a11yCheck requires a context object, and so will not fall back to the document root.
 - .a11yCheck does not return a Promise.
 
+### Virtual DOM Utilities
+
+#### API Name: axe.utils.getFlattenedTree
```

Comment:
However, if we do, then we should point ot the algorithm definition in the HTML spec (see the source code comments for the link)